janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -47,11 +49,25 @@
 
 public class RestElasticSearchClient implements ElasticSearchClient {
 
+
     private static final Logger log = LoggerFactory.getLogger(RestElasticSearchClient.class);
 
+    private static final String REQUEST_TYPE_DELETE = "DELETE";
+
```

Comment:
minor, remove extra lines between the constants, lets make these into a group of constants.